internal/connector/pingone/platform/resources/pingone_key.go

```diff
@@ -14,59 +14,80 @@ var (
 )
 
 type PingOneKeyResource struct {
-	clientInfo *connector.PingOneClientInfo
+	clientInfo   *connector.PingOneClientInfo
+	importBlocks *[]connector.ImportBlock
 }
 
 // Utility method for creating a PingOneKeyResource
 func Key(clientInfo *connector.PingOneClientInfo) *PingOneKeyResource {
 	return &PingOneKeyResource{
-		clientInfo: clientInfo,
+		clientInfo:   clientInfo,
+		importBlocks: &[]connector.ImportBlock{},
 	}
 }
 
+func (r *PingOneKeyResource) ResourceType() string {
+	return "pingone_key"
+}
+
 func (r *PingOneKeyResource) ExportAll() (*[]connector.ImportBlock, error) {
 	l := logger.Get()
+	l.Debug().Msgf("Exporting all '%s' Resources...", r.ResourceType())
 
-	l.Debug().Msgf("Fetching all %s resources...", r.ResourceType())
+	err := r.exportKeys()
+	if err != nil {
+		return nil, err
+	}
+
+	return r.importBlocks, nil
+}
 
-	apiExecuteFunc := r.clientInfo.ApiClient.ManagementAPIClient.CertificateManagementApi.GetKeys(r.clientInfo.Context, r.clientInfo.ExportEnvironmentID).Execute
-	apiFunctionName := "GetKeys"
+func (r *PingOneKeyResource) exportKeys() error {
+	// TODO: Implement pagination once supported in the PingOne Go Client SDK
```

Comment:
Is this one special in some way compared to the other resources?

Reply:
The pagination update for the SDK missed a few resources like this one.